test/05LessonFiveTest.js

```diff
@@ -30,13 +30,16 @@ function fizzBuzz() {
     var array = [];
     for(var i=0; i<100; i++) {
       if (i%3===0&&i%5===0) {
-        array.push("fizzbuzz");
+        i = "fizzbuzz";
+        array.push(i);
       }
       else if (i%3===0) {
-        array.push("fizz");
+        i = "fizz";
+        array.push(i);
       }
       else if (i%5===0) {
-        array.push("buzz");
+        i = "buzz";
```

Comment:
why reassigning the value of i instead of directly pushing "fizz" "buzz" "fizzbuzz"?